documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_Spec_InterfacesPayload.adoc

```diff
@@ -260,14 +260,14 @@ e|https://admin-shell.io/aas/API/3/0/SubmodelServiceSpecification/SSP-001 |Indic
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Read Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-004 |Indicates that the server implemented all details of the Submodel Registry Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-005 |Indicates that the server implemented all details of the Submodel Registry Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/0/AssetAdministrationShellRegistryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Full Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/0/AssetAdministrationShellRegistryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/1/AssetAdministrationShellRegistryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Bulk Profile in version 3.1.
+e|https://admin-shell.io/aas/API/3/0/SubmodelRegistryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Submodel Registry Service Specification Full Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/0/SubmodelRegistryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Submodel Registry Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/1/SubmodelRegistryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Submodel Registry Service Specification Bulk Profile in version 3.1.
 e|https://admin-shell.io/aas/API/3/0/DiscoveryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Discovery Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RepositoryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Repository Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RepositoryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Repository Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/1/DiscoveryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Discovery Service Specification Full Profile in version 3.0.
```

Comment:
- URL says it is version 3.1, text says v3.0 - probably text is wrong
- For `DiscoveryServiceSpecification` there is on URL ending in **SSP-001** and one ending in **SSP-002** while both refer to the `Full` profile, although in different versions (v3.0 vs v3.1). Is this intentional? I would expect for the number not to change with versions. Therefore I guess either then `SSP-00x` number is wrong for one of them or the textual description. See https://github.com/admin-shell-io/aas-specs-api/blob/a72897f9faaddf6129951b6822c5c030e6076f15/documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_Spec_InterfacesPayload.adoc?plain=1#L269

Reply:
Explanation of SSP-002 is not correct, I will fix it. The different versions are intended, as SSP-002 will be added with V3.1.0 (and therefore is also not part of the V3.0.2 Word document).